sphinx/ext/autosummary/generate.py

```diff
@@ -629,7 +640,9 @@ def find_autosummary_in_lines(
             m = toctree_arg_re.match(line)
             if m:
                 toctree = m.group(1)
-                if filename:
+                if base_path is not None and posixpath.isabs(toctree):
+                    toctree = os.path.join(base_path, posixpath.relpath(toctree, "/"))
```

Comment:
Similarly I'd prefer to be explicit that we're looking for a prefix of ``'/'``; semantically this is slightly different from normal path manipulations

Reply:
Done in cdca8ed26.